src/components/input.js

```diff
@@ -240,20 +249,28 @@ const withAutocomplete = WrappedComponent => ({ value, onChange, suggestions: ra
               toggleMenu()
             } else if (_.includes(e.key, ['ArrowUp', 'ArrowDown']) && !suggestions.length) {
               e.nativeEvent.preventDownshiftDefault = true
+            } else if (e.key === 'Enter') {
+              onPick && onPick(value)
```

Comment:
```suggestion
              onPick?.(value)
```

Reply:
I had to revert this change - eslint was complaining about it. It looks like a [known issue](https://github.com/facebook/create-react-app/issues/8107). I tried the two solutions with the babel plugin and typescript version of the rules but it let to a series of other (unwanted) warnings in our code.

Reply:
Gross.